pkg/sql/create_table_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package sql
```

Comment:
should this be in the `types` package and the file named `pkg/sql/types/minimum_type_version_test.go`

Reply:
Done